src/three-components/ModelScene.js

```diff
@@ -126,19 +113,15 @@ export default class ModelScene extends Scene {
     this.isVisible = false;
     this.isDirty = false;
 
-    this.roomBox = new Box3();
-    this.roomSize = new Vector3();
+    this.framedHeight = 1;
```

Comment:
Can we document somewhere what `framedHeight` represents, and how it might change over time? The original concept had a pretty nice comment to this effect.